temp-e2e-test/src/lib.rs

```diff
@@ -0,0 +1,110 @@
+// this is to be removed once the chainspec behaves as expected
+use casper_types::{bytesrepr::Bytes, runtime_args, ContractHash, RuntimeArgs, SecretKey};
+use casper_client::{types::{DeployBuilder, ExecutableDeployItem}};
+
+async fn submit_batch_to_local_net(proof: Proof){
+    let contract_hash: ContractHash = ContractHash::from_formatted_str("contract-cf2e4aa0ea2bb692a3b3decd89b241bc3e3b326b6e964333de7466d3891de29b").unwrap();
+    let runtime_args = runtime_args! {"risc0_receipt" => Bytes::from(serde_json_wasm::to_vec(&proof).unwrap())};
+    let deposit_session: ExecutableDeployItem = ExecutableDeployItem::StoredContractByHash { hash: contract_hash, entry_point: "submit_batch".to_string(), args: runtime_args };
+    let deploy = DeployBuilder::new("cspr-dev-cctl", deposit_session, &SecretKey::from_file("/Users/chef/Desktop/secret_key.pem").unwrap()).with_standard_payment(1000_000_000_000u64).build().unwrap();
```

Comment:
Remove hard coded path.

Reply:
This entire crate is to be deleted.

Reply:
I'm closing this issue since the crate that it addresses has been deleted.